### ISSUE TITLE
[suggestion] Non ES6 Branch

### ISSUE THREAD

**Issue by lezetho** [memte/v14-slash-command-handler#15]
I'm not a ES6 type of guy so I usually avoid projects that have it. I don't avoid this because it's just so good but it takes me a lot of time to convert it all from ES6 to module. It would be great if we could have a non-ES6 branch so people (like me) don't have to spend hours of time converting code into module.

**Comment by memte** [comment]
Yes, that's a bad situation. We've had this before. But since I merged the v13 and v14 repositories, I thought about removing them. I will add it again. Thanks for your suggestion.

**Comment by lezetho** [comment]
How wonderful, thank you so much!

**Comment by lezetho** [comment]
would like a update :D

**Comment by memte** [comment]
Oh. I'm sorry I completely forgot. I'm preparing it now.

**Comment by lezetho** [comment]
No worries, thank you!

**Comment by memte** [comment]
Here:

- https://github.com/memte/ExampleBot/tree/v14-commonjs (v14)
- https://github.com/memte/ExampleBot/tree/v13-commonjs (v13)

**Comment by lezetho** [comment]
Thank you so much!
